Libs/DICOM/Core/ctkDICOMScheduler.cpp

```diff
@@ -834,7 +830,10 @@ void ctkDICOMScheduler::stopJobsByJobUIDs(const QStringList &jobUIDs)
       job->setStatus(ctkAbstractJob::JobStatus::Stopped);
       worker->cancel();
       }
-  }
+    }
+
+  this->waitForDone(2000);
```

Comment:
Why do we wait for done? We should just request stopping (should take no time, just setting a flag in the job), then release the joblist mutex and let things go.

Reply:
already removed in latestt commit